src/api/Nhs.Appointments.Persistance/UserStore.cs

```diff
@@ -19,4 +21,46 @@ public async Task<IEnumerable<RoleAssignment>> GetUserRoleAssignments(string use
         var userDocument = await cosmosStore.GetByIdAsync<UserDocument>(userId);
         return userDocument.RoleAssignments.Select(mapper.Map<RoleAssignment>);
     }
+    
+    public async Task SaveUserAsync(string userId, string scope, IEnumerable<RoleAssignment> roleAssignments)
+    {
+        var documentType = cosmosStore.GetDocumentType();
```

Comment:
This can be moved as it is only used in one branch of the logic. Move it so it is close to the point it is used and in the most appropriate scope